### PR TITLE
[SYTO-1063] Update Installation Documentation for Mortar

### PR DESCRIPTION
**Pull request by asabaylus**
https://jira.natgeo.com/browse/SYTO-1063
```
$ git checkout SYTO-1063 
$ gulp serve
```
Please review the documentation.

### CODE REVIEW

File: src/site/index.html [hunk 1]
```diff
@@ -18,6 +18,41 @@ <h1 class="mt3_h1 intro-txt">Welcome to Mortar</h1>
...
       </p>
+
+
+      <h1 class="mt3_h2 intro-txt">Install Mortar with npm</h1>
+      <p class="mt3_subh1 intro-txt mt3_row-gut-2">
+        If you want a part of Mortar, use our private npm account. First log in into npm.
+      </p>
+
+      <code class="mt3_bgcolor--white mt3_row-gut-2 codemarkup">
+        	$ npm install @natgeo/mortar_v3
```
Comment (pmourelle) [inline src/site/index.html]: How will this be with `mortar_v3` going back to `mortar` (as tickets discussed and added to this sprint). I'm not seeing clear what is "private" in mortar, or why should be consider anything to be private.

Comment (pmourelle) [inline src/site/index.html]: @krlospelaez I think all reference for 'mortar_v3' shouldn't exist as the idea is to have only one 'mortar'. No one outside our closed group should EVER know about the atrocities done to this project naming & versioning. Same as no login should be required (paragraph above) as there are no public repo for it ([wink wink](https://www.youtube.com/watch?v=GkpYqKKrewU)).

File: src/site/index.html [hunk 2]
```diff
@@ -18,6 +18,90 @@ <h1 class="mt3_h1 intro-txt">Welcome to Mortar</h1>
...
+        <br />
+        Individual icons can be found:
+      </p>
+      <code class="mt3_bgcolor--white mt3_row-gut-2 codemarkup">
+        node_modules/@natgeo/mortar_v3/src/icons/
+      </code>
+
+      <h2 class="mt3_h2 intro-txt">Using Mortar Fonts</h2>
+      <p class="mt3_subh1 intro-txt mt3_row-gut-2">
+        Mortar's fonts are not included on the package, however you can...
```
Comment (pmourelle) [inline src/site/index.html]: ... wait for it?

**Comment by asabaylus** [comment]
@pmourelle this should be reconciled with #379

**Comment by pmourelle** [comment]
LGTM

**Pull request merged**